### ISSUE TITLE
box-model rule should not warn if box-sizing is inherited

### ISSUE THREAD

**Issue by cowwoc** [CSSLint/csslint#544]
Given:

```
.moreInfo {
  float: left;
  height: 420px;
  width: 180px;
  padding: 20px;
  border-bottom: 1px solid #ccc;
  border-left: 1px solid #ccc;
  border-right: 1px solid #ccc;
  display: none;
}

* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```

cssLint warns:

`Using height with border-bottom can sometimes make elements larger than you expect. Don't use width or height when using padding or border. (box-model) Browsers: All`

Expected behavior: cssLint should not issue a warning because `box-sizing: border-box` is inherited for `.moreInfo` so the warning does not apply.
